src/base/misc/fatfs.c

```diff
@@ -666,6 +668,8 @@ static char *system_type(unsigned int t) {
         return "Old FreeDOS";
     case FD_D:
         return "FreeDOS";
+    case FDP_D:
+        return "FreeDOS Plugin";
```

Comment:
No, please don't call it freedos. ;)
Its a separate project, and "PP" stands
for "PlusPlus" (because of C++ port).